main.go

```diff
@@ -139,98 +153,114 @@ func exportDSYMs(dsymDir string, dsyms []string) error {
 	return nil
 }
 
-func main() {
-	var cfg configs
-	if err := stepconf.Parse(&cfg); err != nil {
-		fail("Issue with input: %s", err)
+// XcodeVersionProvider ...
+type XcodeVersionProvider interface {
```

Comment:
Is there a specific reason to use exported functions/structs?

Reply:
No